packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2273,6 +2273,16 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         this.dataStores.deleteUnusedRoutes(dataStoreUnusedRoutes);
     }
 
+    /**
+     * When running GC in tombstone mode, this is called to tombstone datastore routes that are unused. This enables
+     * testing scenarios without actually deleting content. The content acts as if it's deleted to the external user,
+     * but the internal runtime acts as if it exists - summarized, etc.
```

Comment:
nit: I wouldn't say "acts as if it exists", maybe just "but it's not actually deleted from summaries"